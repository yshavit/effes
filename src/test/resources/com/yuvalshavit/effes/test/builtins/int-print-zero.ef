getZero -> IntZero: return 0

main:
  print getZero
