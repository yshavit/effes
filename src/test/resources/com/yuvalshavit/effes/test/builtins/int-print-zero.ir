def getZero () -> IntZero:
  ReturnStatement
    IntLiteral
      0
