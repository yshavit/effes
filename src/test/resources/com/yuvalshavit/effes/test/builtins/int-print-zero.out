Your argument was: 0
>> null
